how-tos/providing-images-for-img.rst

```diff
@@ -0,0 +1,143 @@
+Providing images for <img>
+==========================
+
+:author: `pkel <https://github.com/pkel>`_
+
+In this tutorial, you will learn how to create an endpoint for providing images to HTML :code:<img> tags without client side javascript.
+The resulting HTML might look like this:
+
+.. code-block:: html
+
+   <img src="http://host/files/42/cats.jpeg" alt="Cute Kittens"/>
+
+In fact, the presented technique is suitable for providing not only images, but arbitrary files.
+
+We will start with a minimal example that highlights the general concept.
+Afterwards we present are more detailed solution that fixes a few shortcomings of the first approach.
+
+Minimal Example
+---------------
+
+PostgREST returns binary data on requests that set the :code:`Accept: application/octet-stream` header.
+The general idea is to configure the reverse proxy in front of the API to set this header for all requests to :code:`/files/`.
+We will show how to achieve this using Nginx.
+
+First, we need a public table for storing the files.
+
+.. code-block:: postgres
+   create table public.files(
+     id   int primary key
+   , blob bytea
+   );
```

Comment:
```suggestion
.. code-block:: postgres

   create table public.files(
     id   int primary key
   , blob bytea
   );
```
It needs a newline after `code-block` otherwise it doesn't render properly.